chainercv/transforms/bbox/non_maximum_suppression.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import division
+import numpy as np
+
+
+def non_maximum_suppression(bbox, threshold, limit=None, return_param=False):
+    """Suppress bounding boxes according to their Jaccard overlap.
+
+    This method checks each bounding box sequentially and selects the box
+    if the Jaccard overlap between the box and previously selected boxes
+    is less than :obj:`threshold`. This method is mainly used as postprocessing
+    of object detection. In this case, the bounding boxes should be sorted in
+    descending order.
+
+    The bounding boxes are expected to be packed into a two dimensional
+    tensor of shape :math:`(R, 4)`, where :math:`R` is the number of
+    bounding boxes in the image. The second axis represents attributes of
+    the bounding box. They are :obj:`(x_min, y_min, x_max, y_max)`,
+    where the four attributes are coordinates of the bottom left and the
+    top right vertices.
+
+    Args:
+        bbox (~numpy.ndarray): Bounding boxes to be augmented. The shape is
+            :math:`(R, 4)`. :math:`R` is the number of bounding boxes.
+        threshold (float): Thresold of Jaccard overlap.
+        limit (int): the upper bound of the number of selected boxes. If it is
```

Comment:
`the number of boxes to be selected` sounds `the number of the input bounding boxes`. How about `the number of the output boxes`?